apps/site/src/components/pages/blocks/block-form-styles.tsx

```diff
@@ -1,17 +1,17 @@
-import { experimental_sx as sx, Paper, styled } from "@mui/material";
+import { Paper, styled } from "@mui/material";
 
-export const BlockFormContainer = styled(Paper)(
-  sx({ flex: 1, width: "100%", p: 6 }),
+export const BlockFormContainer = styled(Paper)(({ theme }) =>
+  theme.unstable_sx({ flex: 1, width: "100%", p: 6 }),
 );
 
 export const BlockListContainer = styled(BlockFormContainer, {
   shouldForwardProp: (propName) => propName !== "hasBlocks",
 })<{
   hasBlocks?: boolean;
 }>(
-  ({ hasBlocks }) =>
+  ({ theme, hasBlocks }) =>
     hasBlocks &&
-    sx({
+    theme.unstable_sx({
```

Comment:
Good question, previously we were using the `experimental_sx` function here which introduced a TSC compile error in this PR (potentially because the underlying version of `@mui/system` updated with the introduction of the `material-ui-popup-state` package). I looked into the MUI docs and it seems that after [these changes](https://github.com/mui/material-ui/pull/35150) it's now recommended to use `theme.unstable_sx` instead of importing `experimental_sx`, which is why I've made these changes.

Reply:
Thank you